x-pack/test_serverless/api_integration/test_suites/common/security/misc.ts

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FtrProviderContext } from '../../../ftr_provider_context';
+
+export default function ({ getService }: FtrProviderContext) {
+  const svlCommonApi = getService('svlCommonApi');
+  const supertest = getService('supertest');
+
+  describe('security/misc', function () {
+    describe('route access', () => {
+      describe('disabled', () => {
+        it('get index fields', async () => {
+          const { body, status } = await supertest
+            .get('/internal/security/fields/test')
+            .set(svlCommonApi.getInternalRequestHeader())
+            .send({ params: 'params' });
+          svlCommonApi.assertApiNotFound(body, status);
+        });
+
+        it('fix decprecated roles', async () => {
```

Comment:
__typo:__ here and below
```suggestion
        it('fix deprecated roles', async () => {
```